django/db/models/functions/math.py

```diff
@@ -0,0 +1,201 @@
+import math
+
+<<<<<<< HEAD
+from django.db.models import Transform
+
+
+class Abs(Transform):
+	function = 'ABS'
+	lookup_name = 'abs'
+
+class Acos(Transform):
+	function = 'ACOS'
+	lookup_name = 'Acos'
+
+class Asin(Transform):
+	function = 'ASIN'
+	lookup_name = 'Asin'
+
+class Atan(Transform):
+	function = 'ATAN'
+	lookup_name = 'Atan'
+
+class Atan2(Transform):
+	function = 'ATAN2'
+	lookup_name = 'Atan2'
+
+class Ceiling(Transform):
+	function = 'CEILING'
+	lookup_name = 'Ceiling'
+
+class Cos(Transform):
+	function = 'COS'
+	lookup_name = 'Cos'
+
+class Cot(Transform):
+	function = 'COT'
+	lookup_name = 'Cot'
+
+class Degrees(Transform):
+	function = 'DEGREES'
+	lookup_name = 'Degrees'
+
+class Exp(Transform):
+	function = 'EXP'
+	lookup_name = 'Exp'
+
+class Floor(Transform):
+	function = 'FLOOR'
+	lookup_name = 'Floor'
+
+class Log(Transform):
+	function = 'LOG'
+	lookup_name = 'Log'
+	def as_postgresql(self, compiler, connection):
```

Comment:
Please try to follow the indentation style. Same goes for blank lines between the function definitions.

Reply:
Hi atombrella,
I just corrected the codes and reformat them as you suggested. Thank you very much. All failed tests have the same issue: pr/python/python3.5/tests/db_functions/test_math.py", line 5, in <module> from django.db.models.functions import (
ImportError: cannot import name 'PI'. However, I works just fine when I do the test locally.
Do you know what is the problem?
Best,
Junyi